### DISCUSSION TITLE
Feature: Backup settings

### DISCUSSION THREAD

**Discussion by ezxpro** [FireCubeStudios/Protecc#74] [Q&A]
I make periodical backups of my computer and recently I reinstalled windows and want to pull Protecc settings from the old backup image.

For most apps it's as simple as going to the %APPDATA% directory in the backup and copy the corresponding folder, but I can't seem to find the protecc config files.

This would help, otherwise I'll have to set up a VM and run the backup image in it, just to export Protecc configs, it's easier if I can just copy its settings over.

Thanks.

**Comment by ezxpro** [comment]
@FireCubeStudios any way you could provide me a feedback on this topic?
Much appreciated, thanks.

**Reply by FireCubeStudios** [reply]
Protecc like any uwp app stores settings in a settings.dat file which sould be in the local folder. However I don't think there is a way to backup these settings. I will rename this issue to add settings backup as a feature request for the future.

**Reply by ezxpro** [reply]
Well, thanks for the eagerness to help.

Protecc is a great piece of software, very neat and useful.
